File: Ej 9 (Dec to Bin)/Python/Dectobin.py
## DeathbatO

for i in range(9):
    numero = int(input("Digite el numero a convertir: "))
    num = numero
    expo = 0
    bin = 0
    while numero != 0:
        dig = numero % 2
        bin = bin + dig * pow(10, expo)
        expo += 1
        numero = numero//2
    print(f"El numero {num} en binario es: {bin}")

print("Programa Finalizado")

File: Ej 9 (Dec to Bin)/Kotlin/Dectobin.kt
//DeathbatO

import kotlin.math.pow

fun main() {
    for (i in 0 until 9) {
        print("Digite el numero a convertir: ")
        val numero = readLine()?.toInt() ?: 0
        var num = numero
        var expo = 0
        var bin = 0

        var tempNumero = numero
        while (tempNumero != 0) {
            val dig = tempNumero % 2
            bin += dig * 10.0.pow(expo).toInt()
            expo++
            tempNumero /= 2
        }

        println("El numero $num en binario es: $bin")
    }

    println("Programa Finalizado")
}